src/adcs_handler.c

```diff
@@ -35,12 +35,12 @@
  * @return
  * 		Success of function defined in adcs_types.h
  */
-ADCS_returnState adcs_telecommand(uint8_t* command, uint32_t length) {
+ADCS_returnState adcs_telecommand(uint8_t* command, uint32_t length){
   ADCS_returnState ack = ADCS_OK;
 
 #ifdef USE_UART
   ack = send_uart_telecommand(command, length);
-#elif USE_I2C
+#elif defined(USE_I2C)
```

Comment:
I think `defined()` is not necessary, but if it's a more common thing, we should make UART like this too for consistency.

Reply:
I was getting compilation errors without it, for some reason. Apparently, #elif "needed and expression" for the condition. Yes, we could make it like this for UART also.